libiocage/lib/Distribution.py

```diff
@@ -97,6 +101,41 @@ def _filter_available_releases(self, release_name):
         arch = release_name.split("-")[-2:][0]
         return self.host.processor == arch
 
+    def _get_eol_list(self, releases: List[str]) -> List[str]:
+        """Scrapes the FreeBSD website and returns a list of EOL RELEASES"""
+        _eol = "https://www.freebsd.org/security/unsupported.html"
+        req = requests.get(_eol)
```

Comment:
Can we cache this in a file next to `default.json` and check if atime is not older than 3 days (or another reasonable time)? Maybe `eol.json`?

Reply:
That may not work, considering users can disable atime altogether. We should think of a less fragile mechanic. Perhaps we encode the date in the JSON? Check that for 3 days later?

Reply:
we can always write the etag and/or other cache identifying headers into eol.json

Reply:
We could also just use https://pypi.python.org/pypi/CacheControl/0.6 and bypass having a file at all still.